packages/rum/src/domain/record/serialize.spec.ts

```diff
@@ -655,33 +661,46 @@ describe('serializeAttribute ', () => {
     expect(serializeAttribute(node, NodePrivacyLevel.MASK, 'data-foo', DEFAULT_CONFIGURATION)).toBe('***')
   })
 
-  it('does not mask the default programmatic action name attributes', () => {
-    const node = document.createElement('div')
-    node.setAttribute(DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE, 'foo')
-    expect(
-      serializeAttribute(node, NodePrivacyLevel.MASK, DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE, DEFAULT_CONFIGURATION)
-    ).toBe('foo')
-  })
+  describe('attributes used to generate CSS selectors', () => {
+    it('does not mask the default programmatic action name attributes', () => {
+      const node = document.createElement('div')
+      node.setAttribute(DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE, 'foo')
+      expect(
+        serializeAttribute(
+          node,
+          NodePrivacyLevel.MASK,
+          DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE,
+          DEFAULT_CONFIGURATION
+        )
+      ).toBe('foo')
+    })
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I was wondering if we could remove the `DEFAULT_PROGRAMMATIC_ACTION_NAME_ATTRIBUTE` export but it is still used here.
I am assuming it is still interesting to have a test for this case.
I don't mind keeping the export for it since at least we know that we are referencing the same thing.
However, with the same logic, we could maybe also reference `STABLE_ATTRIBUTES` instead of using `'data-test-id'` for the other test:
```ts
it('does not mask other attributes used to generate CSS selectors', () => {
      const node = document.createElement('div')
      node.setAttribute('data-test-id', 'foo')
      expect(serializeAttribute(node, NodePrivacyLevel.MASK, 'data-test-id', DEFAULT_CONFIGURATION)).toBe('foo')
    })
```
wdyt?